src/pages/academia.js

```diff
@@ -49,6 +49,9 @@ export default () => {
         { label: 'MongoDB for Academia', target: '/academia' },
     ];
 
+    const scrollToRef = ref =>
```

Comment:
cool! Looks like this is also [supported on all browsers](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo) so nice touch